library/src/commonMain/kotlin/org/jraf/klibnotion/model/exceptions/NotionClientRequestException.kt

```diff
@@ -56,9 +56,11 @@ private data class NotionError(
     val details: JsonObject? = null,
 ) {
     companion object {
+        private val json = Json { ignoreUnknownKeys = true }
+
         fun fromJsonString(jsonString: String): NotionError? =
             try {
-                Json { ignoreUnknownKeys = true }.decodeFromString(jsonString)
+                json.decodeFromString(jsonString)
             } catch (e: Throwable) {
                 null
```

Comment:
I also suggest to not swallow exceptions here. Let's throw it instead. I can do it with separated PR if you agree

Reply:
Or at least log the exception

Reply:
I think throwing would be a bit too much since we are already in the middle of handling errors. Logging is probably better.

Reply:
Ok, I'll add logging the exception here. Don't you mind if it will be separated pull request? I prefer to keep changes incremental 🙂

Reply:
No I don't mind at all 😊.

I'll merge this one then. Thanks!